src/Files.App/Shell/ShellNewMenuHelper.cs

```diff
@@ -79,7 +80,9 @@ private static Task<ShellNewEntry> ParseShellNewRegistryEntry(RegistryKey key, R
 				!valueNames.Contains("Command", StringComparer.OrdinalIgnoreCase) &&
 				!valueNames.Contains("ItemName", StringComparer.OrdinalIgnoreCase) &&
 				!valueNames.Contains("Data", StringComparer.OrdinalIgnoreCase))
+			{
```

Comment:
Why did you add these curly braces?

Reply:
According to the guideline, we should remove brackets only if if statement body and statement body are both one line.

Reply:
Ok